resume-buddy/src/main/webapp/universities.json

```diff
@@ -21603,6 +21603,6 @@
     "institution": "Zane State College"
   },
   {
-    "institution": "duCret School of Arts"
+    "institution": "Other"
```

Comment:
Did you mean to delete "duCret School of Arts"? Or just to add "Other"?

Reply:
Both. I removed duCret because did not fit alphabetically at the end.